src/common/errors/http-error.ts

```diff
@@ -0,0 +1,12 @@
+export default class HttpError extends Error {
+  public httpStatusCode!: number;
```

Comment:
```suggestion
  public httpStatusCode: number;
```
восклецаельный знак нужен, если мы не объявляем свойство в конструкторе. так что можно вдруг чего скипать в таких кейсах